common/util/random.ts

```diff
@@ -1,7 +1,13 @@
-export const randomString = (length = 12) =>
-  Math.random()
-    .toString(16)
-    .substring(2, length + 2)
+// Returns a cryptographically random hexadecimal string of length `length`
+// (thus containing 4*`length` bits of entropy).
+export const randomString = (length = 12) => {
+  const bytes = new Uint8Array(Math.ceil(length / 2))
+  crypto.getRandomValues(bytes)
+  const hex = Array.prototype.map
+    .call(bytes, (b) => ('0' + (b & 0xff).toString(16)).slice(-2))
```

Comment:
This all makes sense to me except why isn't `b & 0xff` a no-op?

I would have written `bytes.reduce((s, b) => s += ('0' + b.toString(16)).slice(-2), '')`.